v2/azure-arm.yaml

```diff
@@ -1308,6 +1308,51 @@ objectModelConfiguration:
         $generatedConfigs:
           ConnectionString: $.Status.ConnectionString
           InstrumentationKey: $.Status.InstrumentationKey
+    2023-01-01:
+      ActionGroup:
+        $export: true
+        $supportedFrom: v2.4.0
+      AutomationRunbookReceiver:
+        WebhookResourceId:
+          $armReference: true
+      AzureFunctionReceiver:
+        FunctionAppResourceId:
+          $armReference: true
+      LogicAppReceiver:
+        ResourceId:
+          $armReference: true
+    2022-10-01:
+      AutoscaleSetting:
+        $export: true
+        $supportedFrom: v2.4.0
+        TargetResourceUri:
+          $armReference: true
+      MetricTrigger:
+        MetricResourceUri:
+          $armReference: true
+    2018-03-01:
+      MetricAlert:
```

Comment:
Is there no newer API for metricAlert?

Reply:
`2018-03-01` is the only version they've published for MetricAlert